tools/build-example.js

```diff
@@ -0,0 +1,32 @@
+// Builds a single example, passed  as the first argument.
+
+var fs = require("fs");
+var path = require("path");
+var exec = require("child_process").execSync;
+
+function run(command) {
+  // TODO: look into https://www.npmjs.com/package/pidusage to get the Memory usage of the process.
+  const start = performance.now();
+  exec(command, {
+    stdio: "inherit",
+    env: {
+      ...process.env,
+      CI: "true", // Disable spinner even when we have a TTY
+    },
+  });
```

Comment:
The old script makes sure to always call the command from the repo root: https://github.com/hashicorp/terraform-cdk/blob/65e7adb207d540298a53be7622f4fd00610f24c6/tools/build-examples.sh#L5
I think we could supply a `cwd` here to do the same?